tools/TsModelGenerator/Converters/EnumValueConverter.cs

```diff
@@ -9,12 +9,12 @@ public class EnumValueConverter : IMemberNameConverter
         public string Convert(string name, MemberInfo memberInfo)
         {
             // Currently the json converter is not being used when serializing. Uncomment the section below once the EnumName attribute is used for serialization to json. 
-            /*var enumAttribute = (EnumMemberAttribute?)memberInfo.GetCustomAttributes(typeof(EnumMemberAttribute), false).FirstOrDefault();
+            var enumAttribute = (EnumMemberAttribute?)memberInfo.GetCustomAttributes(typeof(EnumMemberAttribute), false).FirstOrDefault();
```

Comment:
Tested this - needed to update a few enum attributes so that the generated values didn't change.

Reply:
was necessary because of how the land act take types were named (with spaces).

Reply:
as long as all the corresponding enums are updated this is good!